packages/python/test/fixtures/00-request-path/probes.json

```diff
@@ -1,6 +1,4 @@
 {
-  "version": 2,
-  "builds": [{ "src": "api/*.py", "use": "@vercel/python" }],
```

Comment:
Do the `00-request-path` and `27-square-brackets` fixtures need a `vercel.json`? It looks like every other fixture have one.

Reply:
No, they don't need one. We could remove vercel.json from most of these if we update the probe paths to poke into `/api`, as well.